actions/templates/install/index.js

```diff
@@ -27,13 +27,14 @@ const CREDENTIAL_FLOW_TYPE_ENTP = 'entp';
  */
 const serializeRequestBody = (params) => {
   // Extracting required properties
-  const { orgId, projectName, description, metadata } = params;
+  const { orgId, projectName, description, metadata, apis } = params;
   // Constructing the serialized object
   return {
     orgId,
     projectName,
     ...(description && { description }), // Include description if it exists
-    metadata // Include metadata object
+    metadata, // Include metadata object
+    ...(apis && { apis }) // Include apis array if it exists
```

Comment:
this is a little weird, if apis is null why not return it.
Just like you have to check if the object has `apis`, so will whoever gets the response. Why not just always return the same type of object having possibly null values.